README.md

```diff
@@ -2,3 +2,14 @@
 
 This repository contains the source code for the [cantera.org](https://cantera.org) website. The
 site is built using the [Nikola](https://getnikola.com) static site generator.
+
+## To build the website
+
+0. Create a virtual environment for Nikola (i.e., using conda or virtualenv)
+1. Clone the Cantera source: `git clone https://github.com/Cantera/cantera.git`
+2. Clone the Cantera Jupyter examples: `git clone https://github.com/Cantera/cantera-jupyter.git`
+3. Clone the Cantera website source: `git clone https://github.com/Cantera/cantera-website.git`
+4. Enter the website repo: `cd cantera-website`
+5. Inside the website repo, install the required packages: `pip install -r requirements.txt`
+6. Build the website: `nikola build`
+7. View the website: `open output/index.html`
```

Comment:
This only works on Mac. Probably better to say `nikola auto -b` which will automatically build the website, watch for file updates, and open a browser.

Reply:
Fixed!